homeassistant/components/device_tracker/__init__.py

```diff
@@ -460,6 +465,46 @@ def async_update(self):
             self._state = STATE_HOME
             self.last_update_home = True
 
+        # lookup vendor string for mac address using api.macvendors.com
+        if not self.vendor:
+            self.vendor = yield from self.get_vendor_for_mac()
```

Comment:
We only need to look up the vendor when we're writing it to a config file. All other times there is no need to hit this API.